src/core/util.js

```diff
@@ -283,4 +285,221 @@ var parseTimeSeriesForC3 = function(graph_data) {
   };
 }
 
-module.exports = { parseDataForC3, parseTimeSeriesForC3, dataApiToC3 }
+/*
+    Takes a multistats object of the following form and 
+    1) flattens it, and
+    2) rounds numeric values 
+    for passing to the DataTable component for rendering:
+
+    {
+      "tasmin_Amon_CanESM2_historical_r1i1p1_19610101-19901231": 
+      {
+        "median": 278.34326171875,
+        "min": 225.05545043945312,
+        "units": "K",
+        "mean": 273.56732177734375,
+        "max": 303.601318359375,
+        "ncells": 8192,
+        "stdev": 22.509726901403784,
+        "run": "r1i1p1"
+      },
+      "tasmin_Amon_CanESM2_historical_r1i1p1_19710101-20001231": 
+      {
+        ...
+      }
+    };
+*/
+var parseBootstrapTableData = function(data) {
+    var flatData = [];
+    var model_count = 0;
+    for (let model in data) {
+        var year_range_re = new RegExp("[0-9]{8}","g");
+        var year_range = [];
+        var lastIndex = 0;
+        while (year_range_re.test(model)){
+            year_range_re.lastIndex = lastIndex;
+            year_range.push(year_range_re.exec(model)[0].slice(0,4));
+            lastIndex = year_range_re.lastIndex;
+        }
+        var period = year_range[0] + " - " + year_range[1];
+        var modelInfo = {
+            "model_period": period,
+            "run": data[model]['run'],
+            "min": +data[model]['min'].toFixed(PRECISION),
+            "max": +data[model]['max'].toFixed(PRECISION),
+            "w_mean": +data[model]['mean'].toFixed(PRECISION),
+            "median": +data[model]['median'].toFixed(PRECISION),
+            "w_stdev": +data[model]['stdev'].toFixed(PRECISION),
+            "units": data[model]['units']
+        };
+        flatData.push(modelInfo); 
+    }
+    return flatData;
+}
+
+/*
+    Helper function for exportTableDataToWorksheet, creates summary rows that appear at the top of the exported worksheet
+*/
+var createWorksheetSummaryCells = function(summary_data, time_of_year) {
```

Comment:
This entire method relates only to a "summary" table, so how about making it read easier by eliminating "summary" from all of the variable names below? The reader *knows* that `num_summary_rows` is the number of rows in the summary table, because there's only one table here. So just call it `num_rows`. Likewise for the rest of the variables in the method.